.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.5.1-node-browsers
+        environment:
+          RAILS_ENV: test
+          PGHOST: localhost
+          PGUSER: magnifier
+      - image: postgres:9.5
+        environment:
+          POSTGRES_USER: magnifier
+          POSTGRES_DB: github-magnifier_test 
+          POSTGRES_PASSWORD: ""
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}-{{ checksum "yarn.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            bundle install --jobs=4 --retry=3 --path vendor/bundle
+            bundle exec yarn
+
+      - save_cache:
+          paths:
+            - ./vendor/bundle
+            - ./node_modules
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}-{{ checksum "yarn.lock" }}
+        
+      # Wait for DB to be available
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      # Database setup
+      - run: bundle exec rake db:create
+      - run: bundle exec rake db:schema:load
+
+      # run tests!
+      - run:
```

Comment:
Can you add RuboCop to these checks?

Reply:
True dat, lint fails can be noisy.  But RuboCop failures are legit.  If a RuboCop failure happens in CI it is something that should trigger a failing build, and thereby has to be addressed before merging the PR to master.  Its great to have this protection backstopped at the CI level for ensuring code quality and consistency.